package.json

```diff
@@ -22,7 +22,9 @@
     "lint": "echo \"No linter specified yet\"",
     "test": "ava",
     "release": "semantic-release",
-    "develop": "ag https://raw.githubusercontent.com/asyncapi/generator/v1.0.0-rc.12/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
+    "develop": "ag https://raw.githubusercontent.com/asyncapi/generator/v1.1.5/test/docs/dummy.yml ./ -o test/output --force-write --watch-template",
+    "//develop:install": "Force ag to install your local html-template source code",
```

Comment:
an interesting approach to document a script, I've never seen it before

Reply:
yeah, it is a good trick ; i take this of my REACT experience ;)